sources/packages/api/src/services/application/application.service.e2e-spec.ts

```diff
@@ -128,20 +133,20 @@ describe("ApplicationService", () => {
       const fakeMSFAANumber = createFakeMSFAANumber(testStudent);
       // Enforce that the MSFAA will be in a valid period.
       fakeMSFAANumber.dateSigned = createDateInMSFAAValidPeriod(-1);
-      const testMSFAANumber = await msfaaNumberRepository.save(fakeMSFAANumber);
       // Create fake application to have the MSFAA associated.
       const fakeApplication = createFakeApplicationInAssessment(testStudent);
       const testApplication = await applicationRepository.save(fakeApplication);
+      fakeMSFAANumber.referenceApplication = fakeApplication;
+      const testMSFAANumber = await msfaaNumberRepository.save(fakeMSFAANumber);
 
       try {
         const savedApplication = await applicationService.associateMSFAANumber(
           testApplication.id,
+          OfferingIntensity.fullTime,
         );
         expect(savedApplication.msfaaNumber).toBeTruthy();
         expect(savedApplication.msfaaNumber.id).toBe(testMSFAANumber.id);
       } finally {
-        await applicationRepository.remove(testApplication);
-        await msfaaNumberRepository.remove(testMSFAANumber);
```

Comment:
Removed this as when the reference application is assigned, we cannot remove the column as its part of cascading in the model